visualize-admin/visualization-tool
# fix: Server-side filtering for versioned dimensions

This PR fixes the server-side filtering of versioned dimensions.

It turns out that the server-side filtering didn't work when dimension was a versioned dimension – we were comparing unversioned filter values with versioned filter values.

### How to test
1. Open `Bathing water quality` dataset.
2. Add coloring by Bathing site.
3. See that the chart works.
4. Follow above steps on other environments (e.g. Prod) to see that no data is returned.
